spring-content-s3/src/main/java/internal/org/springframework/content/s3/config/S3StoreFactoryBean.java

```diff
@@ -26,6 +28,9 @@
 	@Autowired
```

Comment:
Shouldn't this also be annotated with `@Autowired(required=false)`?  What's the difference between Reactive usage to the synchronous one?

Reply:
Yes, I was in the middle of refactoring this.  Should be fixed now